tests/integration2/mixer/scenarios_test.go

```diff
@@ -56,22 +56,27 @@ func testMetric(t *testing.T, ctx component.Repository, label string, labelValue
 	t.Helper()
 
 	mxr := components.GetMixer(ctx, t)
+	env, err := kube.GetEnvironment(ctx)
+	if err != nil {
+		t.Fatalf("Could not get test environment: %v", err)
+	}
+	destinationRuleAll := bookinfo.NetworkingDestinationRuleAll
```

Comment:
Could we just select an appropriate default within the framework if mtls is enabled?

Reply:
Added a function to do it... Please review..